modules/sno-du-subscribing-to-the-operators-needed-for-platform-configuration.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-subscribing-to-the-operators-needed-for-platform-configuration_{context}"]
+= Subscribing to the Operators needed for platform configuration
+
+The subscription provides the location to download the Operators needed for platform configuration.
+Configure the subscription as follows:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: cluster-logging
+    namespace: openshift-logging
+spec:
+    channel: "5.0" <1>
+    name: cluster-logging
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: local-storage-operator
+    namespace: openshift-local-storage
+spec:
+    channel: "4.8" <2>
+    installPlanApproval: Automatic
+    name: local-storage-operator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: performance-addon-operator
+    namespace: openshift-performance-addon-operator
+spec:
+    channel: "4.8" <3>
+    name: performance-addon-operator
+    source: performance-addon-operator
+    sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: ptp-operator-subscription
+    namespace: openshift-ptp
+spec:
+    channel: "4.8" <4>
+    name: ptp-operator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: sriov-network-operator-subscription
+    namespace: openshift-sriov-network-operator
+spec:
+    channel: "4.8" <5>
```

Comment:
Check the channel is it a version as opposed to stable

Reply:
I checked all the chanels:
oc get packagemanifest cluster-logging -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
stable

oc get packagemanifest local-storage-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
stable 
 
oc get packagemanifest ptp-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
stable 

oc get packagemanifest sriov-network-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
stable

Reply:
Changed all to "stable"